docs/SDK.md

```
@@ -126,10 +126,10 @@ aether build
   └─ bundles into a .aex file, SDK types stripped automatically
 ```
 
-## Installation (Planned)
+## Installation
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)